piccolo_api/token_auth/middleware.py

```diff
@@ -84,9 +84,11 @@ class TokenAuthBackend(AuthenticationBackend):
     def __init__(
         self,
         token_auth_provider: TokenAuthProvider = DEFAULT_PROVIDER,
+        excluded_paths: t.List[str] = [],
     ):
         super().__init__()
         self.token_auth_provider = token_auth_provider
+        self.excluded_paths = excluded_paths
```

Comment:
This is a very minor nitpick, but this is marginally better:

```python
    def __init__(
        self,
        token_auth_provider: TokenAuthProvider = DEFAULT_PROVIDER,
        excluded_paths: t.Optional[t.List[str]] = None,
    ):
        super().__init__()
        self.token_auth_provider = token_auth_provider
        self.excluded_paths = excluded_paths or []
```

I try and avoid having lists as default values because it can sometimes lead to bugs (if we mutate that list it effects all instances).

Reply:
Or maybe t.Sequence instead of t.List, as we don't explicitly need it to be a list.

Reply:
OK. I will change that.